### ISSUE TITLE
Writing a new article button doesn't exist!

### ISSUE THREAD

**Issue by aml987** [GavickPro/Magazine-Free-Joomla-Template#6]
![capture](https://cloud.githubusercontent.com/assets/7550942/3074937/e59c7166-e35a-11e3-9d08-f4e912522efe.GIF)
I allow visitors adding new articles, can you check it please? http://www.sycourse.com/high/12/a

**Comment by bkrztuk** [comment]
Please chceck this issue with Beez template, this is probably not template issue but configuration problem.

**Comment by aml987** [comment]
what is Beez template؟  this button was work good in old template.

**Comment by bkrztuk** [comment]
Beez is a default Joomla template…

**Comment by aml987** [comment]
I applied Atomic template http://www.sycourse.com/talk/teachers/9
![capture](https://cloud.githubusercontent.com/assets/7550942/3077001/46e99374-e40d-11e3-85a5-44cb9902ef85.GIF)

**Comment by aml987** [comment]
I think that the link is exist but it's hidden some where because I edited a lot of .css files.

**Comment by bkrztuk** [comment]
Only registered visitors can submit an articles yes? Probably without access details I'm unable to check this.

**Comment by aml987** [comment]
the visitors can submit articles

**Comment by bkrztuk** [comment]
I need to check this on clean template because in your one I can't understand any language phrases. It is regular categories view or K2 view?

**Comment by aml987** [comment]
regular categories, try visiting my site from this link please:
https://translate.google.com/translate?hl=ar&sl=auto&tl=en&u=www.sycourse.com

**Comment by bkrztuk** [comment]
Ok, last test - please go to template directory and rename folder html/com_content to __com_content this allows to check whether it is caused by overrided views or something with configuration

**Comment by aml987** [comment]
I can't do that, ERROR 404 will appear on all pages of my website!

**Comment by bkrztuk** [comment]
It is impossible, you can change any name of the directory in template/html directory without any problems - then Joomla will load just default view. Maybe it is cache issue?

**Comment by aml987** [comment]
I edited com_content folder in the root, I'll edit the template's folder.

**Comment by aml987** [comment]
I renamed it to _com_content

**Comment by bkrztuk** [comment]
and the problem still exist? So it is still missing?

**Comment by aml987** [comment]
No, it's not missing, so the problem was in com_content folder?
how can I fix it and keep the style of magazin template?

**Comment by bkrztuk** [comment]
If it is missing it means it is not template issue but just kind of configuration problem, I have no idea how I can help you because when it is not working on default view it is something with settings not with our product.

**Comment by aml987** [comment]
it is not missing

**Comment by bkrztuk** [comment]
Yes probably it is something related to override views, if it is category view please just back all changes and now try to rename folder com_content/category to detect which view cause this issue.
